src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPConstants.cs

```diff
@@ -24,5 +24,9 @@ internal static class RazorLSPConstants
         public const string VSProjectItemsIdentifier = "CF_VSSTGPROJECTITEMS";
 
         public static readonly Guid RazorActiveUIContextGuid = new Guid("3c5ded8f-72c7-4b1f-af2d-099ceeb935b8");
+
+        public const string LanguageServiceString = "4513FA64-5B72-4B58-9D4C-1D3C81996C2C";
+
+        public static readonly Guid LanguageServiceGuid = new Guid(LanguageServiceString);
```

Comment:
I know this is in `RazorLSPConstants` but I still suggest renaming this to be more specific since we also have contained language identifiers here
```suggestion
        public static readonly Guid RazorLanguageServiceGuid = new Guid(RazorLanguageServiceString);
```